### PR TITLE
displays project tags in projects list view

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10290

## Testing
**URL to test:** 
https://deploy-preview-821--atd-moped-main.netlify.app/

**Steps to test:**
go to the project list view and check the "project tags" field. add or delete tags from a project to see it update!

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by mddilley — APPROVED** [review]
Works perfectly according to the test steps! ✨

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js [hunk 1]
```diff
@@ -35,4 +35,13 @@ export const ProjectsListViewExportConf = {
   project_partner: {
     label: "project_partner",
   },
+  contractors: {
+    label: "contractors",
+  },
+  contract_numbers: {
+    label: "stcontract_numbersring",
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js:42]: ☝🏼

**Review by chiaberry — CHANGES_REQUESTED** [review]
thanks for updating the export config, but there's a typo in the label

**Review by roseeichelmann — APPROVED** [review]
Tested with Netlify, it works as described! 🎃

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js (hunk 1 above)
Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js]: thanks for catching the typo!

**Pull request merged**